build/cmake/libmongoc-static-1.0-config.cmake.in

```diff
@@ -54,3 +54,5 @@ set (MONGOC_INCLUDE_DIRS ${MONGOC_INCLUDE_DIRS} ${BSON_INCLUDE_DIRS})
 
 set_and_check (MONGOC_LIBRARY_DIRS "${PACKAGE_PREFIX_DIR}/lib")
 set (MONGOC_LIBRARY_DIRS ${MONGOC_LIBRARY_DIRS} ${BSON_LIBRARY_DIRS})
+
+set (MONGOC_DEFINITIONS ${BSON_DEFINITIONS})
```

Comment:
Let's do this in libmongoc-1.0-config.cmake.in, too, even though it has no effect yet. Someday libbson-1.0-config.cmake may add to BSON_DEFINITIONS.